pycm/pycm_overall_func.py

```diff
@@ -658,7 +658,6 @@ def macro_calc(item):
     except Exception:
         return "None"
 
-
```

Comment:
A line is deleted here mistakenly.

Reply:
I have made the changes of adding/removing the lines. Thank you.